packages/gp2-components/src/templates/__tests__/DashboardPageBody.test.tsx

```diff
@@ -49,5 +49,12 @@ it('renders news section when there are no pages', () => {
   ).not.toBeInTheDocument();
   expect(
     screen.getAllByRole('heading').map(({ textContent }) => textContent),
-  ).toEqual(expect.arrayContaining(['News Title', 'Tutorial Title']));
+  ).toEqual(expect.arrayContaining(['News Title']));
+  expect(screen.getByTestId('view-news').querySelector('a')).toHaveTextContent(
```

Comment:
Please do not use test IDs, you should inject text and get by that instead. I know this was done on two other PRs of which one has been corrected. The intention is that we should be mimicking users as much as possible in our tests. They can see text on a page the cannot see a random ID in order to create better tests